<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Demo</class>
 <widget class="QWidget" name="Demo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="table"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addRow">
       <property name="text">
        <string>Add Row</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addColumn">
       <property name="text">
        <string>Add Column</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="setNewModel">
       <property name="text">
        <string>Set New Model</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateRandom">
       <property name="text">
        <string>Update Random</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resort">
       <property name="text">
        <string>Resort</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="removeRow">
       <property name="text">
        <string>Remove Row</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeColumn">
       <property name="text">
        <string>Remove Column</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
